Containerfile

```diff
@@ -1,6 +1,7 @@
 FROM registry.fedoraproject.org/fedora:39 AS builder
 RUN dnf install -y git-core golang gpgme-devel libassuan-devel && mkdir -p /build/bib
 COPY bib/go.mod bib/go.sum /build/bib
+RUN go env -w GOPROXY=https://proxy.golang.org,direct
```

Comment:
Non-blocking/optional: Fully supportive of this change, that said in the general case I think I would vote to make this an `ARG GOPROXY=https://proxy.golang.org` instead so it can be overridden by downstream build systems where desired.